src/mixin/PostgresqlMixin.coffee

```diff
@@ -0,0 +1,72 @@
+# Mixin to query and write data to postgresql.
+#
+# Please note that you don't need this mixin when you only want to read data from postgresql. Wrap the
+# query in the QueryStream of `pg-query-stream`.
+#
+# Usage:
+#  * Load the mixin:
+#    ```coffee
+#    { PostgresqlMixin } = require('datapumps/mixins')
+#    ```
+#
+#  * Add the mixin and set the postgresql client:
+#    ```coffee
+#    pump
+#      .mixin PostgresqlMixin postgresqlClient
+#    ```
+#
+#  * Use `.query()` method of the pump in `.process()` and wrap in a QueryStream from `pg-query-stream`
+#    ```coffee
+#    pump
+#      .process (data) ->
+#        @query new QueryStream 'INSERT INTO customer (name, address) VALUES ($1, $2)', [ data.name, data.address ]
```

Comment:
@novaki You are completely correct. The QueryStream is only required when buffering the query in a .from() call. When making regular queries, the regular .query() can be used without the QueryStream wrapper. My newest PR fixes that comment.